### DISCUSSION TITLE
Copilot refund after getting student benefits

### DISCUSSION THREAD

**Discussion by Telumire** [community/community#49552] [Copilot Conversations] [answered]
Hi! After receiving student benefits on my "pro" account, I forgot to cancel my Copilot subscription and only realized after receiving a $10 bill today for this month. I would like to request a refund for said payment. I can provide the name of the student account + show proof privately if needed.

**Comment by Ntshangase** [comment]
Hello Telumire, 

 You can try contacting github support directly at: https://support.github.com/contact?tags=hubberfy_account%2Crr-accounts

**Reply by Telumire** [reply]
Thanks a lot, I couldn't find this link 😅

**Reply by Ntshangase** [reply]
Great, If my response helped you,  You can "mark as answer" so it will be easier for other people to find it when they search similar problems.

And I also get one step closer to getting that galaxy brain badge.

**Reply by Telumire** [reply]
Yes of course ! Thanks again, I got my refund 😀

**Comment by Akash1134** [comment, marked as answer]
Hi Everyone,

We now have the option to self-serve refunds for Copilot. Please head over to the [Help with your GitHub account](https://support.github.com/contact/account?flow=refund_copilot_individual) section and save yourself some time.

Note that, in most cases, the Virtual Agent may be able to offer an immediate refund. But, if your concern is not resolved, you can always use the contact form to get in touch with our billing team.

**Comment by Todd9911** [comment]
I want to cancel my account my account is r
robertbuts81@gmail.com

**Comment by Vikto0004** [comment]
Request for Refund on GitHub Pro Subscription

Hello GitHub Support Team,
I hope this message finds you well.
I am writing to request a refund for the GitHub Pro subscription that I purchased yesterday. After subscribing, I realized that I no longer need the Pro features and have subsequently canceled the subscription today.

Here are the details of my transaction:
GitHub Username: [Vikto0004]
Date of Purchase: [12.06.2024]
Transaction ID: [C992416419345702]

I would greatly appreciate it if you could process a refund for this transaction. Thank you for your understanding and assistance.
